definitions/npm/leaflet_v1.1.x/flow_v0.50.x-/leaflet_v1.1.x.js

```diff
@@ -248,24 +248,34 @@ declare module 'leaflet' {
 
     // Basic Types
 
-    declare class LatLng {
+    declare interface ILatLng {
```

Comment:
Why are you creating an interface???
There is only one implementation of ```LatLng``` and flow make me an error:
```Cannot implement implements [1] with `LatLng` because undefined [2] is incompatible with number [3] in property `alt`. (References: [1] [2] [3])```

Reply:
That's because leaflet can accept plain objects instead of LatLng objects. I can't find a better workaround.